src/key.rs

```diff
@@ -607,6 +632,22 @@ fn to_raw_mode(mode: KeyMode) -> raw::KeyMode {
     }
 }
 
+fn to_raw_flag(flag: KeyFlag)->raw::KeyMode {
```

Comment:
We are not gone need it but in general it is probably better to implement the `From` trait